gcp/rakefiles/vars.rb

```diff
@@ -1,4 +1,5 @@
 require "yaml"
+require "securerandom"
```

Comment:
`require`s in alphabetical order please.

Reply:
Okay.